lib/v2/cbirc/index.js

```diff
@@ -0,0 +1,119 @@
+        description: '公告通知',
+        link: `http://www.cbirc.gov.cn/cn/static/data/DocInfo/SelectDocByItemIdAndChild/data_itemId=925,pageIndex=1,pageSize=18.json`,
+        title: '公告通知',
+    },
+    zcfg:{
+        baseUrl: `http://www.cbirc.gov.cn`,
+        description: '政策法规',
+        link: `http://www.cbirc.gov.cn/cn/static/data/DocInfo/SelectDocByItemIdAndChild/data_itemId=926,pageIndex=1,pageSize=18.json`,
+        title: '政策法规',
+    },
+    zcjd: {
+        baseUrl: `http://www.cbirc.gov.cn`,
+        description: '政策解读',
+        link: `http://www.cbirc.gov.cn/cn/static/data/DocInfo/SelectDocByItemIdAndChild/data_itemId=916,pageIndex=1,pageSize=18.json`,
+        title: '政策解读',
+    },
+    zqyj: {
+        baseUrl: `http://www.cbirc.gov.cn`,
+        description: '征求意见',
+        link: `http://www.cbirc.gov.cn/cn/static/data/DocInfo/SelectDocByItemIdAndChild/data_itemId=951,pageIndex=1,pageSize=18.json`,
+        title: '征求意见',
+    },
+    xzxk:{
+        baseUrl: `http://www.cbirc.gov.cn`,
+        description: '行政许可',
+        link: `http://www.cbirc.gov.cn/cn/static/data/DocInfo/SelectDocByItemIdAndChild/data_itemId=930,pageIndex=1,pageSize=18.json`,
+        title: '行政许可',
+    },
+    xzcf:{
+        baseUrl: `http://www.cbirc.gov.cn`,
+        description: '行政处罚',
+        link: `http://www.cbirc.gov.cn/cn/static/data/DocInfo/SelectDocByItemIdAndChild/data_itemId=931,pageIndex=1,pageSize=18.json`,
+        title: '行政处罚',
+    },
+    xzjgcs:{
+        baseUrl: `http://www.cbirc.gov.cn`,
+        description: '行政监管措施',
+        link: `http://www.cbirc.gov.cn/cn/static/data/DocInfo/SelectDocByItemIdAndChild/data_itemId=932,pageIndex=1,pageSize=18.json`,
+        title: '行政监管措施',
+    },
+    gzlw:{
+        baseUrl: `http://www.cbirc.gov.cn`,
+        description: '工作论文',
+        link: `http://www.cbirc.gov.cn/cn/static/data/DocInfo/SelectDocByItemIdAndChild/data_itemId=934,pageIndex=1,pageSize=18.json`,
+        title: '工作论文',
+    },
+    jrzgyj:{
+        baseUrl: `http://www.cbirc.gov.cn`,
+        description: '金融监管研究',
+        link: `http://www.cbirc.gov.cn/cn/static/data/DocInfo/SelectDocByItemIdAndChild/data_itemId=935,pageIndex=1,pageSize=18.json`,
+        title: '金融监管研究',
+    },
+    tjxx:{
+        baseUrl: `http://www.cbirc.gov.cn`,
+        description: '统计信息',
+        link: `http://www.cbirc.gov.cn/cn/static/data/DocInfo/SelectDocByItemIdAndChild/data_itemId=954,pageIndex=1,pageSize=18.json`,
+        title: '统计信息',
+    }
+};
+
+async function getContent(item) {
+    const response = await got({
+        method: 'get',
+        url: 'http://www.cbirc.gov.cn/cn/static/data/DocInfo/SelectByDocId/data_docId=' + item.docId + '.json',
+    });
+    return response.data.data.docClob;
+}
+
+module.exports = async (ctx) => {
+    const category = ctx.params.category || 'ggtz';
+    const cat = categories[category];
+
+    // 请求集合
+    const response = await ctx.cache.tryGet(cat.link, async () => {
```

Comment:
Take a look at https://github.com/DIYgod/RSSHub/pull/8028
I think you can reduce some overhead